pkg/utils/utils.go

```diff
@@ -237,6 +238,49 @@ func convert(strategy esv1beta1.ExternalSecretConversionStrategy, str string) st
 	return strings.Join(newName, "")
 }
 
+// RevertKeys reverts a secret map into a valid key map as expected by push secrets.
+// Replaces encoded characters back to the original value depending on convert strategy.
+func RevertKeys(strategy esv1alpha1.PushSecretConversionStrategy, in map[string][]byte) (map[string][]byte, error) {
```

Comment:
This doesn't feel like a `revert` 🤔. It's more like a `convert` right...?

Reply:
Oh this is actually a `reverse` not a `revert` right?

Reply:
indeed, I guess I mixed the semantic meaning of the terms, I have replace `revert` by `reverse` in the code and comments.